.gitignore

```diff
@@ -65,6 +65,7 @@ C:\\nppdf32Log\\debuglog.txt
 /config/sandbox_proxy.yml
 /config/cache_store.yml
 /config/google_experiments.yml
+/config/max_allowed_days.yml
```

Comment:
Do we really need another config file for this?

Reply:
Maybe it could be one config file for multiple things related to suspension/deletion of tenants? (as it seems plan name would also end up in config?)

Reply:
> Maybe it could be one config file for multiple things related to suspension/deletion of tenants

It is 😄 

> Do we really need another config file for this?

It is not a need, but what do you propose otherwise? and why would it be better?

Reply:
We have to think about supporting this in https://github.com/3scale/openshift-templates and other deployment scenarios. If I were the customer, the less config files I have to know about, the better.

Reply:
@guicassolato They don't need to know... if they don't configure anything, it just picks the defaults 😄 And this was something requested in https://github.com/3scale/porta/pull/555#discussion_r251497355 and https://github.com/3scale/porta/pull/555#discussion_r251485309 so clearly @hallelujah should give his opinion on this. And not only one of us, but we clearly need to agree in our policy on this from now on... either by default allowing configurations or by default not configuring if not needed... This is something that we need to agree on and not changing disagreeing in different PRs and doing something different in different PRs about this.

Reply:
We need to have safe defaults. If the config is missing then it should be working. Enabling or disabling the feature, whatever, BUT it should boot.

It can be in its own file or inside settings.yml, the important thing is that if the config is not there then we should be able to boot the application.

About rules:

- By safe, my opinion is: Do not change current behaviour. Or if new feature => disable the feature unless explicitly told to. We already apply that to rolling updates anyway

Reply:
> "make it book with an invalid configuration and choose the default instead". That is wrong.

What should we do in this specific scenario? raise an specific error of wrong config for this file? @hallelujah @guicassolato

Reply:
We could raise a specific error, yes. Personally, however, changing the default to an invalid value is exception of the exception and whatever runtime error it's raised would be good for me.

Reply:
From @hallelujah :

> Think about the use case
> what is the value of this for someone booting the app?
> - Does the developer want it right away (by default)?
> - Is it only for saas, for us so we know how to boot it correctly?
> - Is it something that if misconfigured you need the defaults or should just disable because it IS misconfigured?

I said:
> For me disabled is a good idea :+1: Better than defaults and better than not booting at all.
> We should never rely on SaaS-only, more after this PR that we are going to allow non-SaaS to use it

He said:
> but as we need it to be a feature, probably having it like:
- configure it correctly
- otherwise you will see a warning that the feature cannot be used because of misconfiguration

And also:
> my POV is the defaults should not be in the code at all
> no values == not enabled